koku/masu/processor/tasks.py

```diff
@@ -700,3 +701,23 @@ def convert_to_parquet(
         msg = f"Failed to convert the following files to parquet:{','.join(failed_conversion)}."
         LOG.warn(log_json(request_id, msg, context))
         return
+
+
+@app.task(name="masu.processor.tasks.remove_stale_tenants", queue_name="remove_stale_tenants")
+def remove_stale_tenants(self):
```

Comment:
```suggestion
def remove_stale_tenants():
```

Reply:
This change was making the celery beat schedule not fire.  After removing it it ran as expected 👍